keyboards/bastardkb/dilemma/3x5_3/rules.mk

```diff
@@ -6,10 +6,14 @@ MOUSEKEY_ENABLE = yes       # Mouse keys
 EXTRAKEY_ENABLE = yes       # Audio control and System control
 CONSOLE_ENABLE = no         # Console for debug
 COMMAND_ENABLE = no         # Commands for debug and configuration
-NKRO_ENABLE = no            # Enable N-Key Rollover
+NKRO_ENABLE = yes           # Enable N-Key Rollover
```

Comment:
Is this intentional or was this for tests?

Reply:
Intentional, `NKRO_ENABLE = yes` will still default to NKRO off, but in combination with `MAGIC_ENABLE = yes` will allow Via-only users to have the option of toggling NKRO on if they wish, by assigning the magic keycodes.